### PR TITLE
[ADVAPP-497]: Introduce template acceptable use policy for colleges upon tenant creation

### PR DESCRIPTION
**Pull request by danharrin**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-497

### CODE REVIEW

File: app-modules/consent/database/seeders/ConsentAgreementSeeder.php [hunk 1]
```diff
@@ -47,8 +47,84 @@ public function run(): void
...
-                'title' => 'Artificial Intelligence Usage Agreement',
-                'description' => 'Please confirm that you have read the following agreement and consent to the terms and conditions.',
+                'title' => 'Acceptable Use Policy for AI Assistant',
+                'description' => 'Please review and agree to the acceptable use policy (AUP) in order to use this technology.',
+                'body' => <<<EOT
+                PLEASE READ THIS POLICY CAREFULLY. BY CLICKING "I ACCEPT" OR OTHERWISE USING ANY PART OF THIS ENTERPRISE ARTIFICIAL INTELLIGENCE (“AI”) ASSISTANT TOOL, YOU ARE AGREEING TO THE TERMS OF THIS POLICY. If you do not agree to these terms, please do not use this AI Assistant tool.
+
+                Welcome to [Institution Name]'s AI Assistant Tool. This Acceptable Use Policy ("Policy”) governs the use of generative AI assistant tools (“AI Assistant Tool”), including but not limited to the Advising App by Canyon GBS™, by all employees (“You” or “Your”). By using the AI Assistant Tool, You agree to comply with this Policy. Failure to comply may result in disciplinary action, up to and including termination of access and legal consequences.
```
Comment (mxm1070) [inline app-modules/consent/database/seeders/ConsentAgreementSeeder.php:55]: Did we come to a course of action for the `[Institution Name]` placeholder?

File: app-modules/consent/database/seeders/ConsentAgreementSeeder.php [hunk 2]
```diff
@@ -47,8 +47,84 @@ public function run(): void
...
+
+                Acceptable Uses
+
+                1.1. Authorized Users. The AI Assistant Tool is provided for use by [Institution Name] for legitimate institution-related purposes only. The AI Assistant Tool is to be utilized exclusively for professional activities that directly support Your duties at [Institution Name]. The use of these technologies for personal gain or activities outside the scope of Your responsibilities during work hours is strictly prohibited.
+
+                1.2. Professional Conduct. You are expected to use the AI Assistant Tool in a professional manner and in accordance with [Institution Name]’s policies and procedures. [Optional: Insert link to applicable institution policies and procedures here]
```
Comment (mxm1070) [inline app-modules/consent/database/seeders/ConsentAgreementSeeder.php:61]: Same as above.

File: app-modules/consent/database/seeders/ConsentAgreementSeeder.php [hunk 3]
```diff
@@ -47,8 +47,84 @@ public function run(): void
...
+
+                Disclaimer and Notices.
+
+                4.1. Disclaimer. The AI Assistant Tool relies on technology that is still developing, and it may produce unexpected or inaccurate output. By using the AI Assistant Tool, You acknowledge that You are aware of and accept these risks.
+
+                4.2. Inherit Bias Notice. AI systems may inherit biases from the data and Large Language Model (LLM) they are trained on. You are reminded to be vigilant about potential biases in AI-generated content or decision-making processes to avoid unintended discrimination or misrepresentation.
+
+                4.3. Quality Control Notice. Although AI can automate numerous tasks, human oversight remains essential. You bear the responsibility of reviewing and editing AI-generated content to ensure it aligns with the [Institution Name]’s voice, values, and quality standards, maintaining the integrity and reputation of the institution.
```
Comment (mxm1070) [inline app-modules/consent/database/seeders/ConsentAgreementSeeder.php:120]: Same as above.

**Review by mxm1070 — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
@mxm1070 Yeah I discussed with Joe, the user will change it manually.

**Review by mxm1070 — APPROVED** [review]

**Pull request merged**